fl4health/clients/moon_client.py

```diff
@@ -35,10 +38,11 @@ def __init__(
             loss_meter_type=loss_meter_type,
             checkpointer=checkpointer,
         )
-        self.cos_sim = torch.nn.CosineSimilarity(dim=-1).to(self.device)
-        self.ce_criterion = torch.nn.CrossEntropyLoss().to(self.device)
         self.contrastive_weight = contrastive_weight
+        self.mkmmd_loss_weights = mkmmd_loss_weights
         self.temperature = temperature
```

Comment:
Since temperature is just being used to initialize the ContrastiveLoss class, I don't think we need to keep it around as a class variable?

Reply:
That is a fair point